Explorer/Assets/Scripts/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -340,7 +341,7 @@ async UniTask InitializeContainersAsync(IPluginSettingsContainer settingsContain
                 ),
                 new ProfilePlugin(container.ProfileRepository, profileCache, staticContainer.CacheCleaner, new ProfileIntentionCache()),
                 new MapRendererPlugin(mapRendererContainer.MapRenderer),
-                new MinimapPlugin(staticContainer.AssetsProvisioner, container.MvcManager, mapRendererContainer, placesAPIService),
+                new MinimapPlugin(staticContainer.AssetsProvisioner, container.MvcManager, mapRendererContainer, placesAPIService, realmData, container.ChatMessagesBus),
```

Comment:
With main's latest changes we need to update that `realmData` with `staticContainer.RealmData`

Reply:
Done!